qualang_tools/addons/variables.py

```diff
@@ -0,0 +1,12 @@
+from qm.qua import *
+
+
+def assign_variables_to_element(element, *variables):
+    """
+    Forces the given variables to be used by the given element thread. Useful as a workaround for when the compiler
+    wrongly assigns variables which can causes gaps.
+    """
+    _exp = Cast.to_int(variables[0])
+    for variable in variables[1:]:
```

Comment:
Won't this fail if variables is just a single QUA variable?

Reply:
No, it works :)